### PR TITLE
chore(test,ci): fix broken e2e test execution

### PR DESCRIPTION
**Pull request by ScrewTSW**
### What does this PR do?

* Fixes missing playwright dependencies installation

### What issues does this PR fix or reference?
#4983
#5032 

### How to test this PR?

* yarn --frozen-lockfile
* yarn run test:e2e:smoke

### CODE REVIEW

**Review by jeffmaury — APPROVED** [review]
LGTM

**Review by benoitf — CHANGES_REQUESTED** [review]
it only needs to be done once, so it should not be done at each run of the test command

**Comment by odockal** [comment]
@benoitf It does not force overwrite the artifacts, so if it is called twice, it does not do anything if already up to date. 
Before it was part of `yarn install`, now it would be part of `yarn test:e2e:XXX`, which is truly called more than once when running tests, but it is better, imho, than adding this steps to the every workflow.

**Comment by benoitf** [comment]
> so if it is called twice, it does not do anything if already up to date
yes but it's still not something that we should do there



move it to `postinstall` where it means it should be only done 'once' when dependencies are updated/changed

**Comment by odockal** [comment]
@benoitf Nice, neat.

File: package.json [hunk 1]
```diff
@@ -43,8 +43,8 @@
     "compile:current": "cross-env MODE=production npm run build && electron-builder build --config .electron-builder.config.cjs",
     "test": "npm run test:unit && npm run test:e2e",
     "test:unit": "npm run test:main && npm run test:preload && npm run test:preload-docker-extension && npm run test:renderer && npm run test:tools && npm run test:extensions",
-    "test:e2e": "cross-env NODE_ENV=development MODE=development DEBUG=pw:browser npm run build && npm run test:e2e:run",
-    "test:e2e:build": "cross-env NODE_ENV=development MODE=development DEBUG=pw:browser npm run build",
+    "test:e2e": "npm run test:e2e:build && npm run test:e2e:run",
```
Comment (odockal) [inline package.json:46]: As per discussion on PR, please revert running the install part in tests, and put it into a postinstall.

```
...
"prepare": "husky install",
"postinstall": "npx playwright install webkit ffmpeg"
```

**Review by odockal — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (odockal) [reply package.json:46]: @benoitf Like this?

Comment (ScrewTSW) [reply package.json:46]: fixed, tested.

**Review by odockal — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -92,7 +92,8 @@
     "website:prod": "cd website && yarn run docusaurus build && yarn serve",
     "website:dev": "cd website && yarn run docusaurus start",
     "website:screenshots": "cd website-argos && yarn run screenshot",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "postinstall": "npx playwright install webkit ffmpeg"
```
Comment (benoitf) [inline package.json]: why do we need these extra items ( WebKit and ffmpeg) and not only install ?

Comment (odockal) [reply package.json]: `npx playwright install` downloads all default browser, (including chromium, firefox) and it is not necessary, imho.

Comment (benoitf) [reply package.json]: @odockal it does require webkit for electron ? AFAIK electron is based on chromium

Comment (odockal) [reply package.json]: @benoitf I am not sure about what exactly electron requires, but I tried `webkit + ffmpeg` and `chromium` only (it pulls ffmpeg by default) and both combination works. So I guess we can stick to defaults which could be more general for any OS we will run workflow in.

Comment (benoitf) [reply package.json]: @odockal and only ffmpeg ?

Comment (odockal) [reply package.json]: @benoitf Well.. maybe, I will let @ScrewTSW to verify.

**Comment by odockal** [comment]
It is all messy. I have created new issue that this PR directly addresses. It is #5032.
#4983 which I though was to be addressed by this PR will probably not be fixed by this.

File: package.json (hunk 2 above)
Comment (ScrewTSW) [reply package.json]: @benoitf hello, we have reset the requirements to the default ones as they were being downloaded previously

Would that be an acceptable solution?

**Review by benoitf — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -92,7 +92,8 @@
     "website:prod": "cd website && yarn run docusaurus build && yarn serve",
     "website:dev": "cd website && yarn run docusaurus start",
     "website:screenshots": "cd website-argos && yarn run screenshot",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "postinstall": "npx playwright install"
```
Comment (benoitf) [inline package.json]: remove npx before merging

Comment (ScrewTSW) [reply package.json]: done.

**Pull request merged**